source/core/ut_suite_manager.pkb

```diff
@@ -659,10 +692,39 @@ create or replace package body ut_suite_manager is
     return l_suites;
   end;
 
+  function build_no_test_error_msg(
+    a_schema_name in varchar2,
+    a_suite_path in varchar2,
+    a_procedure_name in varchar2,
+    a_object_name in varchar2,
+    a_tags in varchar2    
+    ) return varchar2 is
+    l_error_msg varchar2(500);
+  begin
+    if a_suite_path is not null then 
+      l_error_msg := 'No suite packages found for path '||a_schema_name||':'||a_suite_path;
+    elsif a_procedure_name is not null then 
+      l_error_msg := 'Suite test '||a_schema_name||'.'||a_object_name|| '.'||a_procedure_name||' does not exists';
+    elsif a_object_name is not null then
+      l_error_msg := 'Suite package '||a_schema_name||'.'||a_object_name|| ' does not exists';
+    end if;
+    
+    if l_error_msg is null and a_tags is not null then
```

Comment:
I'm not sure if we should raise if run is empty due to `tag` filters`.
If we execute `ut.run()` we get no errors of there are no suites in schema.
Maybe we should ignore non-existing tags and only filter by those that actually exist?

Reply:
This error is when we are calling a ut.run with a test that doesn't exists. This causing a suite to not be found. 
I think we should do same we do for suitepath e.g. raise an error for non existing suitepath as the tag is an explicit filter on tests we want to execute.
If we dont find any the user should be aware of that.